src/app/(private)/admin-registration-applications/[id]/page.tsx

```diff
@@ -1,8 +1,156 @@
 'use client'
-export default function Page({
-  params,
-}: {
-  params: { slug: string; id: string }
-}) {
-  return <div>{params.id}</div>
+
+import { useEffect, useState } from 'react'
+import { getAdminRegistrationApplication } from '@/modules/adminRegistrationApplications/service'
+import { Input } from '@/components/ui/input'
+import {
+  Select,
+  SelectTrigger,
+  SelectValue,
+  SelectContent,
+  SelectItem,
+} from '@/components/ui/select'
+import { AdminRegistrationApplication } from '../types'
+import { formatDate } from '@/app/hocs/formatDate'
+import Status from '@/modules/adminRegistrationApplications/components/status'
+
+const Page = ({ params }: { params: { slug: string; id: string } }) => {
+  const [
+    adminRegistrationApplicationDetails,
+    setAdminRegistrationApplicationDetails,
+  ] = useState<AdminRegistrationApplication | null>(null)
+  const [loading, setLoading] = useState(true)
+  const [error, setError] = useState<any>(null)
+
+  useEffect(() => {
+    const fetchDetails = () => {
+      getAdminRegistrationApplication(params.id)
+        .then((response) => {
+          console.log(response)
+          if (response.data.isSuccess) {
+            console.log(response.data.response)
+            setAdminRegistrationApplicationDetails(response.data.response)
+          } else {
+            setError(new Error('Failed to fetch data'))
+          }
+          setLoading(false)
+        })
+        .catch((err) => {
+          setError(err)
+          setLoading(false)
+        })
+    }
+    fetchDetails()
+  }, [params.id])
+
+  const renderInput = (label: string, value: string | number | null) => (
+    <div className="w-full">
+      <label className="block text-sm font-medium text-gray-500">{label}</label>
+      <Input value={value ?? ''} disabled className="mt-1 block w-full" />
+    </div>
+  )
+
+  const renderPhoneInput = (countryCode: string, lineNumber: string) => {
```

Comment:
Bu şekilde gerekli olan fonksiyonları modules dosyası içinde ilgili sayfa için açılan dosya içindeki component dosyasına konumlandırmalı ve oradan çekerek kullanmalıyız.